sass/_theme_layout.sass

```diff
@@ -2,6 +2,15 @@
 html
   font-size: $base-font-size
 
+// Set uniform form font styles
```

Comment:
The comment is not needed, setting a light bg and dark text is not something that requires explaining IMHO